code/js/modules/Sitelist.js

```diff
@@ -145,7 +145,7 @@
       "shortorange": { name: "ShortOrange", url: "http://www.shortorange.com" },
       "siriusxm": { name: "SiriusXM", url: "https://player.siriusxm.com" },
       "slacker": { name: "Slacker", url: "http://www.slacker.com" },
-      "somafm": { name: "SomaFM", url: "http://somafm.com" },
+      "somafm": { name: "SomaFM", url: "http://somafm.com", controller: "SomaFMController.js" },
```

Comment:
Not sure if this is a platform specific issue, but for me on windows 10 chrome, the controller failed to load until I made this change.

Reply:
Hmm that is interesting, I'll try to see if I can reproduce that issue locally. Adding the controller property is fine though